projects/transfer-journey/src/lib/components/make-transfer-form.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, Input, OnInit, Output, EventEmitter } from "@angular/core";
+import { AbstractControl, FormBuilder, FormGroup, Validators } from "@angular/forms";
+import { Account, Transfer } from "../model/Account";
+
+@Component({
+  selector: 'bb-make-transfer-form',
+  templateUrl: 'make-transfer-form.component.html'
+})
+export class MakeTransferFormComponent implements OnInit {
+  @Input() account: Account | undefined;
+  @Input() showMaksIndicator: boolean = true;
```

Comment:
Is the name `showMaksIndicator` is intended?

Reply:
yes, it is a name "******" the indicator makes you fill out with asterisc

Reply:
there is a typo showMa**ks**Indicator !== showMa**sk**Indicator

Reply:
I fixed the typo